src/Appwrite/Platform/Tasks/SDKs.php

```
@@ -449,57 +463,95 @@ public function action(?string $platform, ?string $sdk, ?string $version, ?strin
 
                 try {
                     $sdk->generate($result);
                 } catch (\Throwable $exception) {
                     Console::error($exception->getMessage());
                 }
 
+                // Use AI to determine version bump and changelog if _APP_ASSISTANT_OPENAI_API_KEY is set and --ai is not no
+                // This uses the already generated SDK to compare with the remote repo
+                $useAi = ($ai !== 'no');
+                $apiKey = $useAi ? System::getEnv('_APP_ASSISTANT_OPENAI_API_KEY', '') : '';
+                $aiChangelog = ''; // Track AI-generated changelog for PR description
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$aiChangelog'. (undefined)

(UnusedLocalVariable)